### PR TITLE
FIx parsing of MODAPS PDR

### PR DESCRIPTION
**Pull request by cappelaere**
**Summary:** Summary of changes

Addresses [CUMULUS-526: Parsing of MODAPS PDR](https://bugs.earthdata.nasa.gov/browse/CUMULUS-526)

## Changes

* Accept MD5 and checksum as a string
* ...

## Test Plan
Things that should succeed before merging.

- [ ] Unit tests
- [x] Adhoc testing
- [ ] Update CHANGELOG
- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [ ] Commit `.eslint-ratchet-high-water-mark` if the score has improved

Reviewers: @scisco @laurenfrederick

### CODE REVIEW

File: packages/ingest/parse-pdr.js [hunk 1]
```diff
@@ -53,12 +53,12 @@ function parseSpec(pdrName, spec) {
   }
 
   // make sure FILE_CKSUM_TYPE value is CKSUM
-  if (checksumType && checksumType !== 'CKSUM') {
-    throw new PDRParsingError('UNSUPPORTED CHECKSUM TYPE');
+  if (checksumType && (checksumType !== 'CKSUM') && (checksumType !== 'MD5')) {
```
Comment (scisco) [inline packages/ingest/parse-pdr.js]: please add tests for these and update the changelog

**Comment by cappelaere** [comment]
Should be good now

File: tasks/parse-pdr/index.js [hunk 2]
```diff
@@ -78,3 +78,13 @@ function handler(event, context, callback) {
...
+
+// use node index.js local to invoke this
+// justLocalRun(() => {
+//   //const payload = require('@cumulus/test-data/cumulus_messages/parse-pdr.json'); // eslint-disable-line global-require, max-len
+//   const payload = require('/Users/pcappela/Development/gitc-deploy/ingest-sips/messages/parse-pdr.json');
+//   handler(payload, {}, (e, r) => {
+//     //console.log(e);
+//     console.log(JSON.stringify(r, null, '\t'));
+//   })
+// });
```
Comment (abarciauskas-bgse) [inline tasks/parse-pdr/index.js]: Would prefer to remove commented code, and the module import on line 8.

**Comment by scisco** [comment]
@cappelaere the tests are failing

**Pull request merged**